store/errors.go

```diff
@@ -88,3 +90,21 @@ func NewErrNotFound(resource, id string) *ErrNotFound {
 func (e *ErrNotFound) Error() string {
 	return "resource: " + e.resource + " id: " + e.Id
 }
+
+// ErrOutOfBounds indicates that an invalid parameter, usually not related with the entity, was provided
+type ErrOutOfBounds struct {
+	Entity string
+	Args   model.StringInterface
```

Comment:
Is there a reason for these to be exported? Doesn't look like we are accessing these fields from outside.

Reply:
Yep, but other errors here also have exported fields so I wanted to be consistent